great-idea-website/index.html

```diff
@@ -6,58 +6,98 @@
 
   <title>Great Idea!</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Roboto+Mono|Titillium+Web&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
-  <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+<body>
+  <section id='fixed-header'>
+    <nav>
+      <div id='links'>
+          <a href='#'>Services</a>
+          <a href='#'>Product</a>
+          <a href='#'>Vision</a>
+          <a href='#'>Features</a>
+          <a href='#'>About</a>
+          <a href='#'>Contact</a>
+      </div>
+      
+      <div id='logo-image'>
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </div>
+    </nav>
+  </section>
+
+  <header>
+    <div class='headline'>
+        <h1>Innovation</h1>
+        <h1>On</h1>
+        <h1>Demand</h1>
+        <br>
+        <button>Get Started</button>
+        <br>
+        <br>
+        <br>
```

Comment:
Great use of <br>, but for better practice and dry code add a class to the button and play around with the margins and padding next time 🙂